script/eslint-plugin-va/rules/prefer-web-component-library.js

```diff
@@ -9,7 +9,7 @@ const telephoneTransformer = (context, node) => {
   const patternNode = getPropNode(node, 'pattern');
   const notClickableNode = getPropNode(node, 'notClickable');
   const contactNode = getPropNode(node, 'contact');
-  const contactValue = contactNode?.value.expression;
+  const contactValue = contactNode?.value.expression || contactNode?.value;
```

Comment:
The AST for the following has an `expression` node due to the curly braces:

```jsx
<Telephone contact={'12345'} />
```

And the AST for the following doesn't have that.

```jsx
<Telephone contact='12345' />
```

We are seeing more of that second pattern due to a recent linting rule that was turned on.